backend/app.py

```diff
@@ -251,6 +256,22 @@ def status(uuid):
         status=status_code, mimetype='application/json')
     return response
 
+def __delete_after_delay(base_path: str, uuid: str, delay = 300):
```

Comment:
Here we sleep for 300s. This is 5m. In the frontend, we display 10m.
I think we should sleep for around 12m (in order to also give the user the opportunity to download at the last second with a somewhat slow internet speed.
So I propose to change to 12*60 = 720.
```suggestion
def __delete_after_delay(base_path: str, uuid: str, delay = 720):
```

Reply:
sorry saw it to late, will fix it on the dev branch